app/javascript/components/recipes/new/selected-ingredients.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <div class="flex w-80 flex-col">
+    <span class="flex content-center justify-center">Ingredientes seleccionados</span>
+    <div 
+    class="flex p-4 grid grid-cols-2 items-center justify-between "
+      v-for="(element, index) in elements"
+      :key="index"
+      :class="index%2 === 1 ? 'bg-white' : 'bg-gray-100'"
+    >
+      <span class="flex content-center justify-center">
+        {{ element.name }}
+      </span>
+      <button
+       @click="deleteIngredient(index)"
+      >
+        <img
+              class="w-4 h-4"
+              src="../../../../assets/images/cancel-svg.svg"
```

Comment:
⚠️ **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Expected indentation of 10 spaces but found 14 spaces  vue/html-indent